### PR TITLE
Linux a11y sound theme rpm

### PR DESCRIPTION
**Pull request by kk-at-redhat**

### CODE REVIEW

File: ks/vojtux_common.ks [hunk 1]
```diff
@@ -204,9 +207,6 @@ EOM
 chmod 755 /usr/local/bin/monitor-toggle
 echo "Preparing accessibility override..."
 cat > /etc/dconf/db/local.d/01-accessibility <<- EOM
-# Be nice to the users and pre-enable screen reading if they decide to install Gnome.
```
Comment (vojtapolasek) [inline ks/vojtux_common.ks:207]: This dconf section is not related to the sound theme, please keep it in place. It should go into vojtux-settings package which you introduce in a separate PR.

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Hello @kk-at-redhat   and thank you for this contribution.
Please refer to this PR: https://github.com/vojtapolasek/vojtux/pull/48
All my suggestions related to that PR are relevant to this PR as well.

File: ks/vojtux_common.ks (hunk 1 above)
Comment (kk-at-redhat) [reply ks/vojtux_common.ks]: Removed from spec file, added back to common kickstart.

**Comment by kk-at-redhat** [comment]
As requested: added mandatory sections, using install instead of cp where possible, using macros for common system paths. Please, let me know if there is anything else.

File: rpm/a11y-sound-theme/a11y.spec [hunk 2]
```diff
@@ -0,0 +1,34 @@
+Name:     a11y-sound-theme
+Version:  1
+Release:  2
+Summary:  Linux-A11y sound themes
+License:  GPL-3.0
+
+BuildRequires: git
+
+SOURCE1:  12-panel-a11y
```
Comment (vojtapolasek) [inline rpm/a11y-sound-theme/a11y.spec:9]: Hello @kk-at-redhat  and thank you for changes. Actually I have realized one thing - I think we should not force configuration of the sound theme on the user. They should be able to decide if they want to set it as default on their own. At least, that's what I would expect.
Previously this configuration was part of the kickstart because the sound theme was kinda integral part of that kickstart as well... but now when we decided to ship it separately, I think we do not want this.
What do you think?

Comment (kk-at-redhat) [reply rpm/a11y-sound-theme/a11y.spec:9]: I think it comes down to what would be the most comfortable for our target audience, and I'm hardly the right person to give an input on that. If you think it's better not to enable it by default I can go ahead and remove the config part.
From logical point IMHO if we announce a11y is included in our ISO, i.e. it's one of the features, then we might as well enable it and add something in the documentation that explains how to change that.
So just let me know what you want to do.

Comment (vojtapolasek) [reply rpm/a11y-sound-theme/a11y.spec:9]: I think we can do it in a way that everyone is satisfied. I suggest we drop the dconf configuration from the package, but we can keep it in the kickstart file.
My thinking is this: This package can be installed by anyone who subscribes to our copr repo. If I install the package on already installed system, I probably don't expect that it will automatically change my sound theme.
On the other hand, when building Vojtux ISO image, it is fine to have this theme set as default.
Do you agree? Could you please do those changes?

File: rpm/a11y-sound-theme/12-panel-a11y
Comment (vojtapolasek) [inline rpm/a11y-sound-theme/12-panel-a11y:1]: Please rename this file to something else, it has nothing to do with panel. Also please add a new line to the end of the file.

File: rpm/a11y-sound-theme/a11y.spec (hunk 2 above)
Comment (vojtapolasek) [reply rpm/a11y-sound-theme/a11y.spec:9]: AFter gathering fedback from the community and after discussion with @kk-at-redhat  I take back my suggestion of removing the dconf configuration from this package.

File: rpm/a11y-sound-theme/a11y.spec [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+%prep
+
+%build
+git clone https://github.com/coffeeking/linux-a11y-sound-theme.git
+
+%install
+install -d %{buildroot}%{_datadir}/sounds/
+cp -r linux-a11y-sound-theme/linux-a11y %{buildroot}%{_datadir}/sounds/
+install -d %{buildroot}%{_sysconfdir}/dconf/db/local.d
```
Comment (vojtapolasek) [inline rpm/a11y-sound-theme/a11y.spec:22]:
```suggestion
install -d %{buildroot}%{_sysconfdir}/dconf/db/distro.d
```

File: rpm/a11y-sound-theme/a11y.spec [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+install %{SOURCE1} %{buildroot}%{_sysconfdir}/dconf/db/local.d/12-panel-a11y
+
```
Comment (vojtapolasek) [inline rpm/a11y-sound-theme/a11y.spec:24]: Please add following sections as we modify the dconf:
```
%post
dconf update

%postrm
dconf update
```

**Comment by vojtapolasek** [comment]
Hello @kk-at-redhat  I reviewed the PR once more. Please see the suggestions and also please rebase the PR. Thank you.

**Review by vojtapolasek — APPROVED** [review]
